app/views/pizzas/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1 style='margin-top:100px'> Record a new Pizzeria in NYC</h1>
```

Comment:
This header should be dynamic, using erb tags to insert the name of the Pizzeria.